css/main.css

```diff
@@ -1,3 +1,96 @@
 /* Add your custom CSS here */
 
 /* Dont' forget to link this file to your HTML in the <head> */
+
+body {
+    font-family: 'Quicksand', sans-serif;
+    -webkit-font-smoothing: antialiased;
+    position: relative;
+    color:black;
+    margin-left: 15px;
+    margin-right: 15px;
+    
+}
+
+.appointment-button {
+    padding: 5px;
+    border: none;
+    color: black;
+
+}
+ 
+.appointment-button:hover {
+    background-color: #4D7369;
+    color: white;
+    border: none;
+    transition: all 0.7s ease;
+}
+
+.title {
+    font-family: 'Libre Baskerville', serif;
+}
+
+
+ h1 {
+    font-family: 'Libre Baskerville', serif;
+    font-size: 2em + 5vw;
+    color: black;
+ }
+
+h6 {
+    font-family: 'Libre Baskerville', serif;
+}
+
+
+
+.bg-limegreen {
+    background-color:#E2F6CD;
+}
+.bg-lightgrey {
+    background-color:#F2F2F1;
+}
+
+.bg-lightmint {
+    background-color: #CFF4EC;
+}
+
+.bg-mediummint {
+    background-color:#94D6C8;
+}
+
+.bg-darkgreen {
+    background-color: #4D7369;
+}
+
+.about {
+    min-height: 600px;
+}
+
+.hero {
+    min-height: 400px;
+}
+
+.services {
+    min-height: 500px;
+}
+
+.more-services {
+    margin-top: -150px;
+}
+
+.myDIV {
+    transform:translateY(-200px);
```

Comment:
Good work achieving the overlapping image effect 👏 
Try and keep your class name casing consistent - it's common practice to keep CSS class names lower case and separate words with dashes `-`. Also try to use more descriptive names 😉

Reply:
If you want to better control the white spacing below the image, try setting `position: absolute` to it (this will take it out of the flow of the page so the other elements will act as if it wasn't there). Then add some `padding-bottom` to the section so it forces some space for it.